internal/core/testing.go

```diff
@@ -56,7 +56,7 @@ type CheckFuncCtx struct {
 	Result interface{}
 
 	// Meta bag
-	Meta TestMeta
+	Meta metadata
```

Comment:
We have a type Meta that is not use for test. Maybe we should consider renaming this is testMetadata